drivers/overlay/overlay.go

```diff
@@ -1121,7 +1121,7 @@ func (d *Driver) recreateSymlinks() error {
 	// List all the directories under the home directory
 	dirs, err := ioutil.ReadDir(d.home)
 	if err != nil {
-		return fmt.Errorf("error reading driver home directory %q: %v", d.home, err)
+		return fmt.Errorf("reading driver home directory %q: %v", d.home, err)
```

Comment:
```suggestion
		return fmt.Errorf("reading driver home directory %q: %w", d.home, err)
```

`%w` will preserve the error type.

Reply:
There are many other cases below.  Maybe mgrating over to the new go-native error handling should be a different PR?

Reply:
Lets do that in a different PR.